dev/tasks/tasks.yml

```diff
@@ -1435,16 +1435,13 @@ tasks:
   {% endfor %}
 
   test-ubuntu-default-docs:
-    ci: azure
-    template: docker-tests/azure.linux.yml
+    ci: github
+    template: docs/github.linux.yml
     params:
-      artifacts: "build/docs.tar.gz"
```

Comment:
Could you keep this to validate uploaded artifacts (`archery crossbow --validate`) after uploading?

Reply:
isn't line 1445 below for that:
```
    artifacts:
      - docs.tar.gz
```
I think this was unused but I might be confused.
The validation seems correct on the job triggered via crossbow: https://github.com/ursacomputing/crossbow/actions/runs/5643790115/job/15286296284#step:14:24

Reply:
Sorry! You're right!